### PR TITLE
Add additional languages

### PR DESCRIPTION
**Pull request by Peretus**

### CODE REVIEW

**Comment by Peretus** [comment]
Screen Shot 2019-03-11 at 5 53 11 PM

There is one failing test that current requires `re2` that I've not messed with.

File: test/email_reply_parser_test.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@
 require dir + '..' + 'lib' + 'email_reply_parser'
 
 EMAIL_FIXTURE_PATH = dir + 'emails'
+SF_EMAIL_FIXTURE_PATH = dir + 'sf_emails/fr'
```
Comment (ArthurThornton) [inline test/email_reply_parser_test.rb]: Looks like this is unused

File: test/email_reply_parser_test.rb [hunk 2]
```diff
@@ -20,6 +21,36 @@ def test_encoding_should_be_maintained
...
+  end
+  
+  def test_nl_language
+    test_language(:nl)
+  end
+
+  def test_en_language
+    test_language(:en)
+  end
+  def test_it_language
```
Comment (ArthurThornton) [inline test/email_reply_parser_test.rb:50]: Can you add a line between these two method definitions?

**Review by ArthurThornton — CHANGES_REQUESTED** [review]

File: test/email_reply_parser_test.rb (hunk 2 above)
Comment (Peretus) [reply test/email_reply_parser_test.rb:50]: Done!

File: test/email_reply_parser_test.rb (hunk 1 above)
Comment (Peretus) [reply test/email_reply_parser_test.rb]: Oops; not sure why my editor didn't highlight that this time.  Good eye!  Done!

File: lib/email_reply_parser.rb [hunk 3]
```diff
@@ -132,8 +132,37 @@ def read(text)
...
+      /^Le (jeu|lun|ven|dim|mar|sam|mer)\. \d{1,2} \w{3,5}\. .*$/,
+
+      # Nl
+      /^Op (ma|di|wo|do|vr|za|zo) \d{1,2} \w{3,5}\. .*<$/,
+
+      # It
+      /^Il giorno (lun|mar|mer|gio|ven|sab|dom) \d{1,2} \w{3,5}\ .*<$/,
+
+      # Line where the email is split into multiple lines
+      />:$/
```
Comment (Peretus) [inline lib/email_reply_parser.rb:158]: ^ Added this to detect lines that are also considered quoted if they end in `>:`.

File: test/emails/nl/1.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+Message content goes here.
+
+Good to hear from you.
+
+This is still message content.
+
+Op do 22 mrt. 2019 om 13:58 schreef Person (Their Role) <
+conversation+2556@email.example.com>:
+>
```
Comment (Peretus) [inline test/emails/nl/1.txt:11]: ^Added this to the test emails to give us an example of a multi-line quote header.

**Review by ArthurThornton — APPROVED** [review]

**Pull request merged**